HandyHansel/ActionGifs.txt

```diff
@@ -0,0 +1,2 @@
+﻿-- Kill
```

Comment:
Started a list of gifs for later actions that we will have in the bot sort of like Dank Memer or Waifu.